test/benchmarks/datadog/metric_handler.py

```diff
@@ -0,0 +1,167 @@
+from enum import Enum
+import os
+from time import time
+from typing import Dict, List, Optional
+
+from datadog import api, initialize
```

Comment:
I think it would be better to `import datadog`, so in the code we'd write `datadog.api` and `datadog.initialize`. 
It makes it easier to understand what's happening.